server/db.js

```diff
@@ -575,6 +577,39 @@ exports.updateQuestionByQuery = function (query, update, callback) {
     });
 }
 
+// add feedback to feedback collections directly using a query
+exports.addFeedback = function(feedback, callback) {
+    feedbackCollection.insert(feedback, function(err, res) {
+        if (err) {
+            logger.error('Failed to add feedback');
+            return callback(common.getError(8000), null);
+        }
+        
+        return callback(null, 'success');
+    });
+}
+
+exports.getFeedback = function(callback) {
+    feedbackCollection.find().sort({time: -1}).toArray(function(err, res) {
+        if(err) {
```

Comment:
space after `if`

Reply:
I think you mean before if @esteeroramy?